automatic/grafana-agent/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,18 @@
+﻿$ErrorActionPreference = 'Stop';
+$PackageParameters = Get-PackageParameters
+
+$toolsDir     = "$(Split-Path -Parent $MyInvocation.MyCommand.Definition)"
+
+$url          = 'https://github.com/grafana/agent/releases/download/v0.26.1/grafana-agent-installer.exe'
```

Comment:
License allows for redistribution, software is less than 200MB in size and the Grafana package containing the binaries. Please make this package embedded, as the other packages.

Reply:
Was adjusted that the exe is loaded into the package.